src/app/app.module.ts

```diff
@@ -1,21 +1,24 @@
-import {NgModule} from '@angular/core';
-import {BrowserModule} from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
 
```

Comment:
deleting and adding same lines like this might lead to merge conflicts
resolve this